src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplate.utils.ts

```diff
@@ -0,0 +1,78 @@
+
+import { v4 } from 'uuid';
+
+import i18n from 'i18n';
+import { Item, PhrasalTemplateField, PhrasalTemplateFieldType } from 'redux/modules';
+import { ItemResponseType } from 'shared/consts';
+
+// TODO: M2-7211 — Additional response types to be added with
+// additional UI for selecting discrete response values.
+const PHRASAL_TEMPLATE_COMPATIBLE_RESPONSE_TYPES = [
+  ItemResponseType.Date,
+  ItemResponseType.MultipleSelection,
+  ItemResponseType.NumberSelection,
+  ItemResponseType.SingleSelection,
+  ItemResponseType.Slider,
+  ItemResponseType.Text,
+  ItemResponseType.Time,
+  ItemResponseType.TimeRange,
+  // ItemResponseType.ABTrails,
+  // ItemResponseType.Flanker,
+  // ItemResponseType.MultipleSelectionPerRow,
+  // ItemResponseType.SingleSelectionPerRow,
+  // ItemResponseType.SliderRows,
+  // ItemResponseType.StabilityTracker,
+  // ItemResponseType.TouchPractice,
+  // ItemResponseType.TouchTest,
+];
+
+const DEFAULT_PHRASE = {
+  image: '@TODO',
```

Comment:
A default new phrase should just have a `null` image, no? With this `@TODO` here, it's actually breaking the default phrase UI:

![CleanShot 2024-08-01 at 11 26 06@2x](https://github.com/user-attachments/assets/ddd8826c-20ba-44ba-a932-e85c6a2f4b67)

Replacing with `null` seems to make it work, but perhaps there's more needing to be done that was the reason for the `@TODO`?

Also suggest adding explicit typing here:
```suggestion
const DEFAULT_PHRASE: PhrasalTemplateResponseValues['phrases'][number] = {
  image: null,
```